openflow13/match.go

```diff
@@ -473,7 +475,7 @@ func (m *VlanIdField) UnmarshalBinary(data []byte) error {
 }
 
 // Return a MatchField for vlan id matching
-func NewVlanIdField(vlanId uint16) *MatchField {
+func NewVlanIdField(vlanId uint16, mask bool) *MatchField {
```

Comment:
mask needs to be *uint16 type. see NewEthSrcField() for example of how to use masks